frappe/email/doctype/notification/notification.py

```diff
@@ -99,6 +99,9 @@ def on_update(self):
 			if not os.path.exists(path + ".md") and not os.path.exists(path + ".html"):
 				with open(path + ".md", "w") as f:
 					f.write(self.message)
+			if os.path.exists(path + ".md") and not os.path.exists(path + ".html"):
+				with open(path + ".md", "w") as f:
+					f.write(self.message)
```

Comment:
```suggestion
			if not os.path.exists(path + ".html"):
				with open(path + ".md", "w") as f:
					f.write(self.message)
```

Doesn't make sense to check whether the markdown file exists if we're overwriting it regardless

Reply:
Okay @akhilnarang, right 👍 

We have to remove the condition 99–104 and add it which you provide. right?